bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3609,7 +3609,7 @@ void ReplicaImp::executeRequestsInPrePrepareMsg(concordUtils::SpanWrapper &paren
     } else {
       LOG_DEBUG(CNSUS, "Consensus reached");
     }
-
+    SCOPED_MDC("pp_msg_cid", ppMsg->getCid());
```

Comment:
this should also be logged when creating the batch

Reply:
Done.